### DISCUSSION TITLE
Copy 64gb image to 512gb sd, how to expand file system

### DISCUSSION THREAD

**Discussion by rgroothuis** [marcone/teslausb#783] [General]
For a test setup I’ve used a 64gb SD card. Now everything is working fine I want to upgrade to a 512gb SD card. I want to make an image of the 64gb SD card and put it on the 512gb. How can I expand the file system so that the entire SF card is used for the CAM recordings?

**Comment by marcone** [comment]
There are now 4 partitions on the sd card, and you would need to expand the 3rd partition. It's probably possible, but it's likely easier to just use your current teslausb_setup_variables.conf to set it up from scratch on the new sd card.

**Reply by rgroothuis** [reply]
Ok, thanks for the feedback. So the best way forward is to copy the teslausb_variables.conf file to the new 512g SD card on which i’ve installed a fresh image. That is probably the safest way. The raspberry with the teslausb is away right now, so cannot check, but where can i find the conf file?

**Reply by marcone** [reply]
The conf file is moved to /root/teslausb_variables.conf during setup.
